README.md

```diff
@@ -70,17 +73,18 @@ If you want to contribute but are unsure how to start, we have [a "good first is
 
 #### Technologies
 
-With the ground rules out of the way, let's talk about the coarse architecture of this mono repo:
+Let's discuss the key technologies that power this monorepo, providing a foundation for our healthcare application development:
+
+- **Full-stack TypeScript**: Our codebase is primarily built using TypeScript, enabling type-safe development across both the server and frontend components. Node.js is used for server-side code, while React powers the frontend applications.
```

Comment:
This change reads a lot better to me, and does a good job explaining more of the rationale for TypeScript

Reply:
I agree, cheers! _